tests/shared/UserLogonRequestTestDefs.js

```diff
@@ -12,9 +12,9 @@
 
 "use strict";
 
-var fluid = require("infusion"),
+var fluid  = require("infusion"),
```

Comment:
Clownspacing has arisen

Reply:
I protest my innocence, as I didn't directly touch this file.  The extra space is due to merging with @the-t-in-rtf 's GPII-3619 branch *before* it went into master.  See:  https://github.com/GPII/universal/blob/ab5bee75070a3815ea96d2a09c1b58407f63a9e3/tests/shared/UserLogonRequestTestDefs.js#L17  @cindyli noted a few similar issues above.

Perhaps the definition of "Clownspacing" is the act of copying other author's mistakes before they make a final cleanup for merging into master. I can live with that.

Reply:
Please still remove these extra spaces at line 15 and 17. Thanks.

Reply:
Done.